pylintrc

```diff
@@ -0,0 +1,61 @@
+[MASTER]
+ignore = migrations, management
+load-plugins = pylint_django
+
+[MESSAGES CONTROL]
+disable=
+    invalid-name,
+    django-not-configured,
+    consider-using-with,
+    bad-option-value,
+    too-many-positional-arguments,
+    logging-format-interpolation,
+    missing-module-docstring,
+    abstract-method,
```

Comment:
```suggestion
    django-not-configured,
    bad-option-value,
    logging-format-interpolation,
    missing-module-docstring,
    abstract-method,
```
What do you think?

Reply:
Any particular reasons for the above suggestion? do we want to allow `consider-using-with`, `too-many-positional-arguments` exceptions?

Reply:
I disabled these because they cause trouble in some scenarios especially `too-many-positional-arguments`

Reply:
I think we should enable these and enforce good practices. We can always explicitly ignore if there is a real need. What do you think?

Reply:
Sounds well!